requirements-dev.txt

```diff
@@ -1,6 +1,6 @@
 ruff
 cairosvg
-codespell
+codespell>=2.2.6
```

Comment:
For some reason, `codespell` was giving me fits about only wanting to run on Python 3.7 locally, and pinning it here seemed to help :shrug: